### PR TITLE
New package: AtlantisWordProcessor.TheAtlantisWordProcessorTeam version 4.3.7.3

### PR DESCRIPTION
**Pull request by Jcoth**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/145994)

### CODE REVIEW

**Comment by Jcoth** [comment]
@microsoft-github-policy-service agree

File: manifests/a/AtlantisWordProcessor/TheAtlantisWordProcessorTeam/4.3.7.3/AtlantisWordProcessor.TheAtlantisWordProcessorTeam.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: AtlantisWordProcessor.TheAtlantisWordProcessorTeam
+PackageVersion: 4.3.7.3
+PackageLocale: en-US
+Publisher: The Atlantis Word Processor Team
+PackageName: Atlantis Word Processor
+License: Copyright (c) The Atlantis Word Processor Team
```
Comment (Trenly) [inline manifests/a/AtlantisWordProcessor/TheAtlantisWordProcessorTeam/4.3.7.3/AtlantisWordProcessor.TheAtlantisWordProcessorTeam.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

**Review by Trenly — CHANGES_REQUESTED** [review]
Please add the required Installer Switches needed to make this program install unattended
image

File: manifests/a/AtlantisWordProcessor/TheAtlantisWordProcessorTeam/4.3.7.3/AtlantisWordProcessor.TheAtlantisWordProcessorTeam.locale.en-US.yaml (hunk 1 above)
Comment (Jcoth) [reply manifests/a/AtlantisWordProcessor/TheAtlantisWordProcessorTeam/4.3.7.3/AtlantisWordProcessor.TheAtlantisWordProcessorTeam.locale.en-US.yaml]: ok changed to proprietary, i'm sorry and new to this

**Review by Jcoth — COMMENTED** [review]
ok changed to proprietary, i'm sorry and new to this

**Pull request closed without merging**